tests/suites/cli/use_local_charm.sh

```diff
@@ -1,49 +1,122 @@
 # Checks whether the cwd is used for the juju local deploy.
-run_deploy_local_charm_revision() {
+test_deploy_local_charm_revision() {
```

Comment:
Can we keep these as `run_` - these are officially sub tests for `test_local_charms` test, so they shouldn't confuse people

Reply:
Oh sry was not on purpose, probably a rebase error by me 👯‍♂